packages/api/src/routes/medical/patient.ts

```diff
@@ -280,12 +279,14 @@ router.get(
     const cxId = getCxIdOrFail(req);
     const patientId = getFrom("params").orFail("id", req);
     const patient = await getPatientOrFail({ cxId, id: patientId });
-    const respPayload: QueryProgressFromSDK = {
-      status: patient.data.consolidatedQuery?.status ?? null,
-      message:
-        "Trigger a new query by POST /patient/:id/consolidated/query; data will be sent through Webhook",
-    };
-    return res.json(respPayload);
+    const consolidatedQueries = patient.data.consolidatedQueries ?? null;
+    const message =
+      "Trigger a new query by POST /patient/:id/consolidated/query; data will be sent through Webhook";
+
+    return res.json({
+      statuses: consolidatedQueries,
+      message,
+    });
```

Comment:
We are now sending an array of statuses so they can understand where each one is at